examples/official/fashion_mnist_tf_keras/distributed.yaml

```diff
@@ -3,8 +3,8 @@ hyperparameters:
   global_batch_size: 32
   dense1: 128
 resources:
-  # 2-machine distributed run assumes 8 GPUs per agent instance
-  slots_per_trial: 16
+  # 4-machine distributed run assumes 1 GPU per agent instance
+  slots_per_trial: 4
```

Comment:
I don't think we should be should be modifying this, it's rather silly to run single-GPU distributed training jobs.

Reply:
@brain-good I think the idea is to reduce the number of GPUs per machine so that first-time users just trying out Determined won't have to pay for 8GPUs. What if we leave a comment in the distributed.yaml to tell them to re-deploy with more GPUs per machine to get the benefit of distributed?

Reply:
I agree with @brain-good that making 1-GPU boxes the default doesn't seem right. Maybe we should try to instead expand documentation on how users can adjust the default settings for cost savings?

Reply:
Reverted the distributed examples. Added documentation to AWS and GCP install to how to update the cluster for multi gpu training.